src/System.Console/tests/WindowAndCursorProps.cs

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using Xunit;
+
+public class WindowAndCursorProps
+{
+    [Fact]
+    [PlatformSpecific(PlatformID.AnyUnix)]
+    public static void WindowWidth()
+    {
+        Assert.Throws<PlatformNotSupportedException>(() => Console.WindowWidth = 100);
+
+        // Validate that Console.WindowWidth returns some value in a non-redirected o/p.
+        RunInNonRedirectedOutput((data) => Console.WriteLine(Console.WindowWidth));
```

Comment:
CI system will always run these tests with a redirected o/p. And so RunInNonRedirectedOutput would still be running in redirected mode unless we manually test it by explicitly running the corerun command.